src/search/utils/collections.h

```diff
@@ -151,6 +152,98 @@ int estimate_unordered_map_bytes(int num_entries) {
     // See comments for _estimate_hash_table_bytes.
     return _estimate_hash_table_bytes<std::unordered_map<Key, Value>>(num_entries);
 }
+
+    /**
+     * NamedVector is a vector-like collection with optional names to be associated with each element.
+     * It is optimized to have minimal overhead when there are no names.
+     * Any name which is not specified is assumed to be an empty string.
```

Comment:
This comment doesn't follow our convention. See "Comments" under http://www.fast-downward.org/ForDevelopers/CodingConventions